tests/Form/InputWeekTest.php

```
@@ -0,0 +1,601 @@
+        self::assertSame(
+            <<<HTML
+            <input id="inputweek-" type="week" form="form-id">
+            HTML,
+            InputWeek::tag()->form('form-id')->id('inputweek-')->render(),
+            "Failed asserting that element renders correctly with 'form' attribute.",
+        );
+    }
+
+    public function testRenderWithGlobalDefaultsAreApplied(): void
+    {
+        SimpleFactory::setDefaults(InputWeek::class, ['class' => 'default-class']);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Core\Factory\SimpleFactory' in method 'testRenderWithGlobalDefaultsAreApplied'. (undefined)

(StaticAccess)